components/automatic-points-table/commons/automatic-points-table.lua

```diff
@@ -298,4 +299,90 @@ function AutomaticPointsTable:addPositionData(pointsData)
 	)
 end
 
+function AutomaticPointsTable:generatePointsTable(pointsData, tournaments)
+	local columnCount = self:countColumns(tournaments)

```

Comment:
```suggestion
	local columnCount = Array.reduce(tournaments, function(t) return columnCount + (t.shouldDeductionsBeVisible and 2 or 1) end)

```

Double-check the calculation

Reply:
Where does the reduce function come from? do I need to import it?

Reply:
```suggestion
local columnCount = Array.reduce(tournaments, function(count, t)
			return count + (t.shouldDeductionsBeVisible and 2 or 1)
		end)
```